### DISCUSSION TITLE
Search icon not displaying properly?

### DISCUSSION THREAD

**Discussion by nkeil** [NvChad/NvChad#2079] [Q&A] [answered]
I've tried a few different nerd fonts (Hack, Meslo, JetBrains), but nothing will get this search bar icon to display.
<img width="549" alt="image" src="https://github.com/NvChad/NvChad/assets/26151586/f59e5e6a-dcfd-4c74-acf5-7916cad51589">

I assume it's supposed to look like this:
<img width="214" alt="image" src="https://github.com/NvChad/NvChad/assets/26151586/46ddbe8d-e85e-47c4-a8f9-671004c8bcb2">

Any ideas?

**Comment by soomtong** [comment]
do you use iterm2? try other terminal. in my wezterm
<img width="464" alt="image" src="https://github.com/NvChad/NvChad/assets/906974/043274c2-939e-439b-954c-78e2fa3bb1b8">

**Reply by nkeil** [reply]
That works! I was using iterm2. Though, using wezterm, now some other icons are missing, like the folder icon
<img width="183" alt="image" src="https://github.com/NvChad/NvChad/assets/26151586/2b1fa115-fd0e-4316-98c1-39a1f642b3f0">

**Reply by siduck** [reply, marked as answer]
@nkeil Hi,  use latest nerdfonts i.e v3.0.1

**Reply by nkeil** [reply]
@siduck I've tried installing the nerdfonts from from the website, as well as the latest release on github ([link](https://github.com/ryanoasis/nerd-fonts/releases/tag/v3.0.1)), and I'm getting the same results

**Reply by siduck** [reply]
@nkeil make sure to delete old fonts! rm -rf your old fonts!

**Comment by nkeil** [comment]
Just thought I'd follow up here in case anyone else stumbles on this. I came back to this issue to try to get it fixed. Wezterm was still not working for me, and updating my fonts wasn't working either. There was always something missing in either case.

What finally worked was installing the [kitty terminal](https://sw.kovidgoyal.net/) and using their fonts. Everything looks great now.
